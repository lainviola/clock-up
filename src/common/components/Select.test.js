import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Select } from './Select'

describe('<Select />', () => {
  it('toMatchSnapshot', () => {
    const wrapper = shallow(
      <Select>
        <option value="1">1</option>
      </Select>
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
