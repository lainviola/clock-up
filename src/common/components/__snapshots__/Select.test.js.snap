// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select /> toMatchSnapshot 1`] = `
<select
  className="Select-eZBaHc lmGDrE"
>
  <option
    value="1"
  >
    1
  </option>
</select>
`;
